controllers/external/util.go

```diff
@@ -39,6 +39,9 @@ const (
 
 // Get uses the client and reference to get an external, unstructured object.
 func Get(ctx context.Context, c client.Client, ref *corev1.ObjectReference, namespace string) (*unstructured.Unstructured, error) {
+	if ref == nil {
+		return nil, errors.Errorf("Cannot get object - object reference not set")
```

Comment:
```suggestion
		return nil, errors.Errorf("cannot get object from an empty reference")
```

We usually start error messages lowercase, log lines with uppercase